src/utilities/include/sparse.h

```diff
@@ -71,33 +68,33 @@ class SparseIndices {
     stable_sort(autoPolicy(size()), view.begin(), view.end());
   }
 
-  void Resize(int size) { data_.resize(size * sizeof(int64_t), -1); }
+  void Resize(size_t size) { data_.resize(size * sizeof(int64_t), -1); }
 
-  inline int& Get(int i, bool use_q) {
+  inline int& Get(size_t i, bool use_q) {
```

Comment:
These look like the critical changes to fix the bug, right?

Reply:
yes, but not only here because we need to make sure that code in `boolean3.cpp` handles this correctly.